fairroot/geobase/FairGeoCone.cxx

```diff
@@ -40,10 +40,11 @@
 #include <TArrayD.h>     // for TArrayD
 #include <TMathBase.h>   // for Abs
 #include <TString.h>     // for TString
+#include <fmt/core.h>    // for format
 #include <fstream>
 #include <ostream>    // for fstream, etc
 #include <stdio.h>    // for printf, sprintf, sscanf
-#include <string.h>   // for strlen
+#include <string.h>   // for string, strlen
```

Comment:
* `std::string` is in `<string>` (https://en.cppreference.com/w/cpp/string/basic_string)
* `std::strlen` is in `<cstring>` (https://en.cppreference.com/w/cpp/string/byte/strlen)

`<string.h>` is a C std library header (and declares `::strlen` https://en.cppreference.com/w/c/string/byte/strlen)